lib/stud/task.rb

```diff
@@ -36,5 +36,14 @@ def stop!
     def self.interrupted?
       Thread.current[:stud_task_interrupted]
```

Comment:
https://github.com/jordansissel/ruby-stud/issues/25 @colinsurprenant beat me to create the issue.